LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan/DevAddr.cs

```diff
@@ -27,6 +27,15 @@ namespace LoRaWan
 
         public DevAddr(uint value) => this.value = value;
 
+        public DevAddr(int networkId, int networkAddress)
+            : this((networkId, networkAddress) switch
+            {
+                ( > 0x7F, _) => throw new ArgumentException(null, nameof(networkId)),
```

Comment:
What about negative values? Think the following would be better:

```suggestion
                ( < 0 or >= 0x80, _) => throw new ArgumentException(null, nameof(networkId)),
```

Reply:
Done in 01e2056508a2ea255e01e447bf07fce5aa20ecc4